configs/local/core/dashd.conf

```diff
@@ -20,12 +20,9 @@ zmqpubhashblock=tcp://0.0.0.0:29998
 
 # JSONRPC
 server=1
-rpcport=20002
 rpcuser=dashrpc
 rpcpassword=password
 
-rpcallowip=172.27.0.0/16
-rpcallowip=127.0.0.0
```

Comment:
We do not allow requests from the local machine?

Reply:
Added a rule to accept RPC from the host machine

Reply:
Where? :)

Reply:
https://github.com/dashevo/mn-bootstrap/pull/150/files#diff-617ba09d0095f6311d3e3ae245069bd8R22